### PR TITLE
Remove registration

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
I think it's time to remove the registration journey

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

**Review by kicferk1 — APPROVED** [review]
One thing I like about this journey is how easy it is to burn it away.

File: app/controllers/registrations/user_profile_controller.rb [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-    @user = User.new(user_params)
-
-    ActiveRecord::Base.transaction do
-      @user.save!
-      @school.induction_coordinator_profiles.destroy_all
-      InductionCoordinatorProfile.create!(user: @user, schools: [@school])
-    end
-
-    session.delete(:school_id)
-    redirect_to :registrations_verification_sent
```
Comment (kicferk1) [inline app/controllers/registrations/user_profile_controller.rb:21]: Do we have some cypress helpers / mailer stuff that needs to go too?

And maybe some templates in notify?

**Review by amazimbe — APPROVED** [review]
LGTM!

File: app/controllers/registrations/user_profile_controller.rb (hunk 1 above)
Comment (ethanmills) [reply app/controllers/registrations/user_profile_controller.rb:21]: Good shout

Comment (ethanmills) [reply app/controllers/registrations/user_profile_controller.rb:21]: That's a big change, I'll remove in another PR

**Pull request merged**